connect/runtime/src/test/java/org/apache/kafka/connect/runtime/rest/resources/ConnectorsResourceTest.java

```diff
@@ -66,12 +66,26 @@
     private static final String LEADER_URL = "http://leader:8083/";
     private static final String CONNECTOR_NAME = "test";
     private static final String CONNECTOR2_NAME = "test2";
+    private static final String CONNECTOR_NAME_ALL_WHITESPACES = "     ";
```

Comment:
to get a bit better coverage on type of whitespace instead of just spaces, might be worth throwing in a few other whitespace characters into these examples.

Reply:
Will do